packages/react-components/token-analyzer-preview/library/src/astAnalyzer.ts

```diff
@@ -72,15 +72,50 @@ function processStyleProperty(prop: PropertyAssignment, isResetStyles?: Boolean)
           processNode(childProp.getInitializer(), [...path, childName]);
         }
       });
-    } else if(Node.isCallExpression(node)){
+    } else if (Node.isCallExpression(node) && node.getExpression().getText() === 'createCustomFocusIndicatorStyle') {
+      const focus = `&[data-fui-focus-visible]`;
```

Comment:
`data-fui-focus-visible` is what gets placed on the element itself right? Should we instead just use the CSS selector since that would keep it scoped to CSS vs the DOM?